codes/run.py

```diff
@@ -123,7 +124,7 @@ def read_triple(file_path, entity2id, relation2id):
     triples = []
     with open(file_path) as fin:
         for line in fin:
-            h, r, t = line.strip().split('\t')
+            h, r, t = map(lambda x: x.strip(), unicodedata.normalize('NFKC', line).split('\t'))
```

Comment:
`str.strip` could be a shorter replacement for `lambda x: x.strip()`

Reply:
Thanks, didn't realize this was possible.